src/main/java/lotto/PurchaseLotteryTicket.java

```diff
@@ -0,0 +1,84 @@
+package lotto;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PurchaseLotteryTicket {
+    public static final int MATCH_THREE = 3;
+    public static final int MATCH_FOUR = 4;
+    public static final int MATCH_FIVE = 5;
+    public static final int MATCH_SIX = 6;
+    private List<LotteryTicket> purchaseLotteryTicket;
```

Comment:
```suggestion
    public static final int MATCH_SIX = 6;
    
    private List<LotteryTicket> purchaseLotteryTicket;
```

클래스 변수와 인스턴스 변수 사이에 공백을 추가한다면 가독성이 상승합니다 ~

Reply:
네 클래스 변수와 인스턴스 사이 공백 추가! 알려주셔서 감사합니다 :)